projects/nae-example-app/src/app/doc/public-single-task-view/public-single-task-view.component.ts

```diff
@@ -138,12 +144,20 @@ export class PublicSingleTaskViewComponent extends AbstractSingleTaskViewCompone
         );
     }
 
+    ngOnInit(): void {
+        this._router.routeReuseStrategy.shouldReuseRoute = () => false
+    }
+
     ngAfterViewInit(): void {
         this.initializeHeader(this.taskHeaderComponent);
     }
 
     logEvent(event: TaskEventNotification) {
-        console.log(event);
+        if (!!event && event.event === TaskEvent.FINISH) {
```

Comment:
can be simplified to `event?.event === TaskEvent.FINISH`

Reply:
Sure, I will correct it. Thanks!

Reply:
I just checked, and this part is not even needed.